src/module/actor/party/sheet.ts

```diff
@@ -433,6 +433,10 @@ class PartySheetPF2e extends ActorSheetPF2e<PartyPF2e> {
         htmlQuery(html, "[data-action=rest]")?.addEventListener("click", (event) => {
             game.pf2e.actions.restForTheNight({ event, actors: this.actor.members });
         });
+
+        htmlQuery(html, "[data-action=prompt]")?.addEventListener("click", () => {
+            game.pf2e.gm.skillSavePrompt({ actors: this.actor.members });
```

Comment:
It seems actors isn't respected by the skillSavePrompt. It should either default to `game.actors.party.members` in the skill save prompt, or allow omitting actors altogether.